java/arcs/android/storage/handle/AndroidHandleManager.kt

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright 2020 Google LLC.
- *
- * This code may only be used under the BSD style license found at
- * http://polymer.github.io/LICENSE.txt
- *
- * Code distributed by Google as part of this project is also subject to an additional IP rights
- * grant found at
- * http://polymer.github.io/PATENTS.txt
- */
-package arcs.android.storage.handle
-
-import android.content.Context
-import androidx.lifecycle.Lifecycle
-import arcs.core.storage.StoreManager
-import arcs.core.storage.handle.HandleManager
-import arcs.jvm.util.JvmTime
-import arcs.sdk.android.storage.ServiceStoreFactory
-import arcs.sdk.android.storage.service.ConnectionFactory
-import kotlin.coroutines.CoroutineContext
-import kotlin.coroutines.EmptyCoroutineContext
-
-/**
- * AndroidHandleManager will create a [HandleManager] instance, replacing the default
- * [ActivationFactory] with one that generates [ServiceStore] instances that can
- * communication with a running [StorageService].
- */
-@Deprecated("Use arcs.core.host.EntityHandleManager")
-@OptIn(kotlinx.coroutines.ExperimentalCoroutinesApi::class)
-fun AndroidHandleManager(
```

Comment:
does this mean you cannot create handles in Android without using an archost?

Reply:
You can, just use
```
val androidHandleManager = EntityHandleManager(
  time = JvmTime,
  activationFactory = ServiceStoreFactory(context, lifecycle)
)
```